action.yml

```diff
@@ -4,6 +4,14 @@ inputs:
   token:
     description: github token to clone / push with
     required: false
+  changed:
+    description: only run against changed files in this commit.  requires advanced checkout.
+    required: false
+    default: "false"
+  base:
+    description: base git ref for 'changed' parameter.  requires advanced checkout.
+    required: false
+    default: "master"
 runs:
-  using: 'node12'
-  main: 'dist/index.js'
+  using: "node12"
+  main: "dist/index.js"
```

Comment:
just a tip for open source for the future, it's generally better to follow the existing style than change it -- especially when it's completely unrelated to your patch

Reply:
Yes, missed this left from my editor.